var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -231,8 +231,8 @@ def cmake_args(self):
                 if feature in target:
                     options.append('-DGMX_SIMD:STRING={0}'.format(flag))
                     break
-        else:
-            # Fall back to this for unknown microarchitectures
+
+            # Fall back
             options.append('-DGMX_SIMD:STRING=None')
```

Comment:
When I compile on X86_64 skylake, I get 
```
Command line:
  gmx_mpi mdrun -ntomp 4 -nsteps 1 -s /fsx/benchMEM.tpr

Compiled SIMD: None, but for this host/run AVX_512 might be better (see log).
```
Will send a patch with additional condition for fallback.

Reply:
See #21727